### ISSUE TITLE
Missing visible keyboard focus in post elements (wall entry)

### ISSUE THREAD

**Issue by krallowski** [humhub/humhub#8398]
Several elements have no visible keyboard focus on the Dashboard page:
In a seperate post entry (or feed)
·	heading
·	icon on the left leading to the space
·	menu icon on the top right (three little dots)
·	Opening the dropdown (three little dots): the individual elements
·	Date
·	Image in the post
·	reaction-icons(comment, like and share)
On the right sidebar
·	menu icon on the top right (three little dots)
·	Seperate element of activities

### What steps will reproduce the problem?
Tab through the dashboard and check any visual indicators for the focus.

### What is the expected result?
While tabbing a visual indicator shows which element is focused like underlined or a border.

### What do you get instead?
No visual indicator.

### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   | 1.18.3
| PHP version      | 
| Operating system |
